src/core/reactor.cc

```diff
@@ -2965,6 +2967,35 @@ void reactor::add_urgent_task(task* t) noexcept {
     }
 }
 
+void reactor::run_in_background(future<> f) {
+    if (f.available()) {
+        if (f.failed()) {
+            std::rethrow_exception(f.get_exception());
```

Comment:
once scheduled, it can no longer return the error, just log that it happened and wasn't handled.
it's debatable if we want to throw if the gate is closed.

Reply:
this part removed in v2, so the error will be handled (and ignored) as if it was asynchronous